pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -68,10 +69,11 @@ class LightningKokkos(LightningQubit):
 
     name = "PennyLane plugin for Kokkos-backed Lightning device"
     short_name = "lightning.kokkos"
-    pennylane_requires = ">=0.22"
+    pennylane_requires = "<=0.28"
```

Comment:
Should we change this before merging? Marking as a reminder.

Reply:
yes, that's what Lee mentioned in our standup last Friday.